core/templates/pages/collection-player-page/collection-local-nav/collection-local-nav.component.html

```diff
@@ -1,6 +1,6 @@
 <ul class="nav navbar-nav oppia-navbar-nav float-right oppia-navbar-pencil-icon">
-  <li ng-if="$ctrl.canEdit" class="nav-item">
-    <a ng-href="/collection_editor/create/<[$ctrl.collectionId]>" uib-tooltip="Edit" tooltip-placement="bottom" target="_blank" rel="noopener" class="nav-link">
+  <li *ngIf="canEdit" class="nav-item">
+    <a ng-href="'/collection_editor/create/' + collectionId" uib-tooltip="Edit" tooltip-placement="bottom" target="_blank" rel="noopener" class="nav-link">
       <i class="material-icons">&#xE254;</i>
       <span class="oppia-icon-accessibility-label">Edit</span>
     </a>
```

Comment:
Wrap everything but style tag in a div with class collection-local-nav and in line number 10, change `collection-local-nav` to `.collection-local-nav`